docs/pages/guides/typescript.mdx

```diff
@@ -85,14 +85,17 @@ You can now run `npm run ts:check` or `yarn tsc` to type-check the project.
 
 A project's **tsconfig.json** should extend the `expo/tsconfig.base` by default. This sets the following default [compiler options](https://www.typescriptlang.org/docs/handbook/compiler-options.html) (which can be overwritten in your project's **tsconfig.json**):
 
+You can automatically generate a **tsconfig.json** file by running: `npx expo customize tsconfig.json`
+
 ## Project configuration
 
 Expo CLI will automatically modify your **tsconfig.json** to the preferred default which is optimized for universal React development:
 
 ```json tsconfig.json
 {
   "extends": "expo/tsconfig.base",
-  "compilerOptions": {}
+  "compilerOptions": {},
+  "include": ["**/*.ts", "**/*.tsx", ".expo/types/**/*.ts", "expo-env.d.ts"]
```

Comment:
As per our discussion and the context you gave to me, can we mention here that `"expo-env.d.ts"` is only added when Expo Router library is installed in the project?